src/inventorymanager.h

```diff
@@ -183,6 +183,18 @@ struct IMoveAction : public InventoryAction, public MoveAction
 	void apply(InventoryManager *mgr, ServerActiveObject *player, IGameDef *gamedef);
 
 	void clientApply(InventoryManager *mgr, IGameDef *gamedef);
+
+	void swapDirections();
+
+	const void onPutAndOnTake(const ItemStack &src_item, ServerActiveObject *player);
```

Comment:
What I meant was `void foo() const;`, declaring any return type `const void` or `const int` is pointless.

Reply:
I also meant `void foo() const;` - but I wrote `const void foo()`

(I'm coming from a Java background where function/method modifiers are writting in front of the function and don't apply to the return type :P)